server/src/main/java/org/candlepin/controller/CandlepinPoolManager.java

```diff
@@ -382,6 +383,9 @@ void refreshPoolsForMasterPool(Pool pool, boolean updateStackDerived, boolean la
                     subPool.setDerivedProduct(update);
                 }
             }
+            if (pool.isLocked()) {
```

Comment:
Whitespace between this block and the previous block

Reply:
Fixed.